src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,29 @@
+.catalog {
+  display: grid;
+  grid-template-columns: 200px;
+  justify-content: center;
+  
+  gap: 48px;
+
+  &__card {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 300px;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+  }
+
+  @media screen and (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media screen and (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media screen and (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+
```

Comment:
You still have empty line before closing tag....

Reply:
Sorry, I didn't see this comment early, done)